Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM python:3.7-alpine
+FROM python:3.9-alpine
```

Comment:
What about using 3.10-alpine? Is it because we don't officially support it yet?

Reply:
Exactly, didn't test with 3.10, but once tests are updated we should update this

Reply:
You can go ahead with 3.10 now that #921 was merged

Reply:
Alright !